src/storage/index.ts

```diff
@@ -0,0 +1,39 @@
+import { MMKV } from 'react-native-mmkv';
+
+type Value = string;
+
+export enum StoreKeys {
+  session = 'session',
+}
+
+export * from './parser';
+
+class Storage {
+  #Store = new MMKV();
```

Comment:
I'm curious about this, do you have a documentation/link? To look at it.

Reply:
yeah sure, @juanchoperezj @tamarasuarezrod 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields